src/Directory.Build.props

```diff
@@ -5,8 +5,6 @@
     <AnalysisLevel>latest-recommended</AnalysisLevel>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <WarningsAsErrors>Nullable</WarningsAsErrors>
```

Comment:
Enforcing nullability warnings as errors seems desirable to me from a modelling perspective. Any chance we could keep this turned on?

Reply:
I see, you want those as errors even in your inner loop? Sure, I can revert that.